source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTextBlockRenderer.mm

```diff
@@ -81,11 +81,8 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
 
         // Obtain text color to apply to the attributed string
         ACRContainerStyle style = lab.style;
-        ColorsConfig colorConfig;
-        if (style == ACREmphasis)
-            colorConfig = config->GetContainerStyles().emphasisPalette.foregroundColors;
```

Comment:
need to change to have reference here. I didn't realize that you've changed this in earlier commit.